synapse/handlers/pagination.py

```diff
@@ -561,9 +572,12 @@ async def get_messages(
                 state_dict = await self.store.get_events(list(state_ids.values()))
                 state = state_dict.values()
 
-        aggregations = await self._relations_handler.get_bundled_aggregations(
-            events, user_id
-        )
+        if not use_admin_priviledge:
+            aggregations = await self._relations_handler.get_bundled_aggregations(
+                events, user_id
+            )
+        else:
+            aggregations = None  # TODO: an admin might want aggregations
```

Comment:
Yeah, I think we should always bundle the aggregations. The `user_id` is merely used to filter out ignored users, so it's safe to just pass in the admin user ID.